### DISCUSSION TITLE
[@types/node] - Unable to Build Node.JS Project after update

### DISCUSSION THREAD

**Discussion by elliot-huffman** [DefinitelyTyped/DefinitelyTyped#63272] [Issues with a @types package]
Hi Team,

After updating to the latest TypeScript `4.9.3`, I got the below `TSC` error.
According to the TypeScript [release announcement](https://devblogs.microsoft.com/typescript/announcing-typescript-4-9/#lib-d-ts-updates), the embedded `lib.d.ts` got updated and now this is now conflicting with this `@types/node` package.

Would you all be able to update the types package so that it no longer is conflicting with the TypeScript 4.9.x version?

Thanks!

Error details:
```
node_modules/@types/node/globals.d.ts:72:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; abort(reason?: any): AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; }'.

72 declare var AbortSignal: {
               ~~~~~~~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:2071:13
    2071 declare var AbortSignal: {
                     ~~~~~~~~~~~
    'AbortSignal' was also declared here.


Found 1 error in node_modules/@types/node/globals.d.ts:72
```

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
@elliot-huffman does it happen if you remove/reinstall @types/node? The recent versions contain TS4.9 specific declaration of this to match updated lib.dom interface (@types/node ships with TS4.9 compatible implementaiton first, and <=TS4.8 as complementary types for older TS version)

**Reply by haines** [reply]
Updating @types/node worked for me. I got this error with 18.7.16 but after updating to 18.11.9 it is no longer an issue.

**Comment by elliot-huffman** [comment]
That fixed it for me!
Thanks much!
